deployments/dev-local-3nodes/docker-compose.yml

```diff
@@ -133,16 +157,12 @@ services:
     extends:
       file: ../docker-compose-definitions.yml
       service: medco-unlynx
-    ports:
-      - "2005:2005"
-      - "2006:2006"
     environment:
       - MEDCO_NODE_IDX=2
       - UNLYNX_DEBUG_LEVEL=3
     networks:
       intra-node-srv2:
       inter-nodes:
-        ipv4_address: 172.31.0.121
```

Comment:
So we removed this specific address because the unlynx nodes no longer talk to each others. They talk directly to the reverse proxy. This `ipv4_address: 172.31.0.121` has been moved to the nginx container.